### PR TITLE
Migrate to styled components

### PR DESCRIPTION
**Pull request by jkettmann**
Migrates home page to styled-components.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -22,10 +66,10 @@ const Home: NextPage = () => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main className={styles.main}>
-        <h1 className={styles.title}>
+      <Main>
+        <Title className={styles.title}>
```
Comment (mentor-tara) [inline pages/index.tsx]: Please remove the `className` here

File: pages/index.tsx [hunk 2]
```diff
@@ -22,10 +66,10 @@ const Home: NextPage = () => {
...
           Welcome to <a href="/about">Prolog</a>
-        </h1>
+        </Title>
 
         <p className={styles.description}>
```
Comment (mentor-tara) [inline pages/index.tsx:74]: Why didn't you migrate all the elements to styled-components?

**Review by mentor-tara — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (jkettmann) [reply pages/index.tsx]: Good catch 👍

File: pages/index.tsx (hunk 2 above)
Comment (jkettmann) [reply pages/index.tsx:74]: Come on. Cut me some slack. This is REALLY boring!

**Review by mentor-tara — APPROVED** [review]
💯

**Pull request merged**